api/resources_portal/test/dev_data/materials.json

```diff
@@ -0,0 +1,377 @@
+{
+  "materials": {
+    "0": {
+      "title": "Melanoma Reduction Plasmid",
+      "material_type": "plasmid",
+      "id": "1",
+      "url": "https://www.addgene.org/53246/",
+      "contact_user": "1",
+      "organization_id": "1",
+      "additional_metadata": {
+        "basic_info": {
+          "plasmid_name": "pGP1871",
+          "plasmid_type": "Encodes one insert",
+          "purpose": "Reduces melanoma growth rate",
+          "organism": "Danio rerio",
+          "Number of Availible Samples": "20",
+          "biosafety_level": "2"
+        },
+        "gene_insert": {
+          "gene_insert_name": {
+            "1": "Danio rerio",
+            "2": "Dani devrito"
+          },
+          "relevant_mutations": "Skin regrowth sequence modified",
+          "plasmid_backbone_name": "pSB1A30",
+          "primary_vector_type": "Bacterial Expression",
+          "cloning_method": "Unknown",
+          "sequence_maps": {
+            "1": "https://bucket-name.s3.region.amazonaws.com/keyname"
+          }
```

Comment:
Sequence Maps are associated with a Sequence (which is a required field) 
Example Sequence:
`TCGCGCGTTTCGGTGATGACGGTGAAAACCTCTGACACATGCAGCTCCCGGAGACGGTCACAGCTTGTCT
GTAAGCGGATGCCGGGAGCAGACAAGCCCGTCAGGGCGCGTCAGCGGGTGTTGGCGGGTGTCGGGGCTGG
CTTAACTATGCGGCATCAGAGCAGATTGTACTGAGAGTGCACCATATGGTACCAAATTTATTGATGGATC
AATTTAAAGAGTTTTAATCTGGGTGTTATGAGTTCCTTGGGCCTATTGTTTGCCTGAACCCTGTGGGGAC
TGGCTCATCAGCAGAATTATTCATGGAGGAATTTGCTAGGTTTTACCTTGGCTCTCTAGCTTGGGACATT
TTGTTTCTTCCTTAAATCCTTATTGCAACCGTCCAGCTACCAGGTAGGTATATTGCCTTGTAAGTGCTGG
TACAGTTTGCCTTATTTATATTCCACTGCTTCTCAGGGATTAACATCTGCTCGTGCAGCTGAGATCCTGG
CGCGAGATGGTCCCAACGCCCTCACTCCCCCTCCCACTACTCCTGAATGGATCAAGTTTTGTCGGCAGCT
CTTTGGGGGGTTCTCAATGTTACTGTGGATTGGAGCGATTCTTTGTTTCTTGGCTTATAGCATCAGAGCT
GCTACAGAAGAGGAACCTCAAAACGATGACGTGAGTTCTGTAATTCAGCATATCGATTTGTAGTACACAT
CAGATATCTTCTCCGTCTTTGTCTCCCACTTCTTCTCAATTACCACTCATTACTTAATGGTTATGAACTC
ATTACTTAATGGTTATGAACAGCTGTTGCCTTCAAGGCTCATCCATTCTTCCTTCGTTTCCATTTCCTCT
CTCTACCACCCACGTTGTAGATGCTCTTACAAGTGGGATGCCCACCTGCATGTGCTGCTGTGAGCAGGCA
GCTCTGCTCAGGCCCCGGCCGCCACCCACTGGATGGCAGAGCACAGCGATTCATGTTGGCACATCCACCT
GTCCAGAAATGCTTGGTGGCCATTCTTCAAAATCCACAAGTTGGTTGAAAAACAATCTTTGTTTTATAAA
GCAGGAGAAACTGATGCATCTAGAACCTTTCCAAACGTCCAGTTAGTGATCAAGTGTTGGTGTGCCTGAC
TCCATTTCTGACCCTTCCTGTGTGGTCTTTAGAAGGGAATTCTGCATTAATGAATCGGCCAACGCGCGGG
GAGAGGCGGTTTGCGTATTGGGCGCTCTTCCGCTTCCTCGCTCACTGACTCGCTGCGCTCGGTCGTTCGG
CTGCGGCGAGCGGTATCAGCTCACTCAAAGGCGGTAATACGGTTATCCACAGAATCAGGGGATAACGCAG
GAAAGAACATGTGAGCAAAAGGCCAGCAAAAGGCCAGGAACCGTAAAAAGGCCGCGTTGCTGGCGTTTTT
CCATAGGCTCCGCCCCCCTGACGAGCATCACAAAAATCGACGCTCAAGTCAGAGGTGGCGAAACCCGACA
GGACTATAAAGATACCAGGCGTTTCCCCCTGGAAGCTCCCTCGTGCGCTCTCCTGTTCCGACCCTGCCGC
TTACCGGATACCTGTCCGCCTTTCTCCCTTCGGGAAGCGTGGCGCTTTCTCATAGCTCACGCTGTAGGTA
TCTCAGTTCGGTGTAGGTCGTTCGCTCCAAGCTGGGCTGTGTGCACGAACCCCCCGTTCAGCCCGACCGC
TGCGCCTTATCCGGTAACTATCGTCTTGAGTCCAACCCGGTAAGACACGACTTATCGCCACTGGCAGCAG
CCACTGGTAACAGGATTAGCAGAGCGAGGTATGTAGGCGGTGCTACAGAGTTCTTGAAGTGGTGGCCTAA
CTACGGCTACACTAGAAGAACAGTATTTGGTATCTGCGCTCTGCTGAAGCCAGTTACCTTCGGAAAAAGA
GTTGGTAGCTCTTGATCCGGCAAACAAACCACCGCTGGTAGCGGTGGTTTTTTTGTTTGCAAGCAGCAGA
TTACGCGCAGAAAAAAAGGATCTCAAGAAGATCCTTTGATCTTTTCTACGGGGTCTGACGCTCAGTGGAA
CGAAAACTCACGTTAAGGGATTTTGGTCATGAGATTATCAAAAAGGATCTTCACCTAGATCCTTTTAAAT
TAAAAATGAAGTTTTAAATCAATCTAAAGTATATATGAGTAAACTTGGTCTGACAGTTACCAATGCTTAA
TCAGTGAGGCACCTATCTCAGCGATCTGTCTATTTCGTTCATCCATAGTTGCCTGACTCCCCGTCGTGTA
GATAACTACGATACGGGAGGGCTTACCATCTGGCCCCAGTGCTGCAATGATACCGCGAGACCCACGCTCA
CCGGCTCCAGATTTATCAGCAATAAACCAGCCAGCCGGAAGGGCCGAGCGCAGAAGTGGTCCTGCAACTT
TATCCGCCTCCATCCAGTCTATTAATTGTTGCCGGGAAGCTAGAGTAAGTAGTTCGCCAGTTAATAGTTT
GCGCAACGTTGTTGCCATTGCTACAGGCATCGTGGTGTCACGCTCGTCGTTTGGTATGGCTTCATTCAGC
TCCGGTTCCCAACGATCAAGGCGAGTTACATGATCCCCCATGTTGTGCAAAAAAGCGGTTAGCTCCTTCG
GTCCTCCGATCGTTGTCAGAAGTAAGTTGGCCGCAGTGTTATCACTCATGGTTATGGCAGCACTGCATAA
TTCTCTTACTGTCATGCCATCCGTAAGATGCTTTTCTGTGACTGGTGAGTACTCAACCAAGTCATTCTGA
GAATAGTGTATGCGGCGACCGAGTTGCTCTTGCCCGGCGTCAATACGGGATAATACCGCGCCACATAGCA
GAACTTTAAAAGTGCTCATCATTGGAAAACGTTCTTCGGGGCGAAAACTCTCAAGGATCTTACCGCTGTT
GAGATCCAGTTCGATGTAACCCACTCGTGCACCCAACTGATCTTCAGCATCTTTTACTTTCACCAGCGTT
TCTGGGTGAGCAAAAACAGGAAGGCAAAATGCCGCAAAAAAGGGAATAAGGGCGACACGGAAATGTTGAA
TACTCATACTCTTCCTTTTTCAATATTATTGAAGCATTTATCAGGGTTATTGTCTCATGAGCGGATACAT
ATTTGAATGTATTTAGAAAAATAAACAAATAGGGGTTCCGCGCACATTTCCCCGAAAAGTGCCACCTGAC
GTCTAAGAAACCATTATTATCATGACATTAACCTATAAAAATAGGCGTATCACGAGGCCCTTTCGTC`